uaa/src/main/resources/uaa.yml

```diff
@@ -405,9 +405,9 @@ login:
     #Local/SP metadata - requests signed
     signRequest: true
     #Local/SP metadata - want incoming assertions signed
-    #wantAssertionSigned: true
+    wantAssertionSigned: true
     #Algorithm for SAML signatures. Defaults to SHA1.  Accepts SHA1, SHA256, SHA512
-    #signatureAlgorithm: SHA256
+    signatureAlgorithm: SHA256
```

Comment:
this changes the default for SAML signatures

Reply:
Yes, we acknowledge that. Do you think that we should not change this?

Reply:
No fine for all... SHA1 would be a risk / issue nowadays

Reply:
Ah, yes, I forgot about this. I was thinking about the other previous PR.